packages/web/src/layouts/swap/components/swap-card/swap-card-content/SwapCardContent.tsx

```diff
@@ -164,7 +177,11 @@ const SwapCardContent: React.FC<ContentProps> = ({
         <div className="amount-container">
           <input
             id={tokenB?.priceID}
-            className={`amount-text ${isLoading && direction === "EXACT_IN" ? "text-opacity" : ""}`}
+            className={`amount-text ${
+              (isLoading && direction === "EXACT_IN") || (isRefetching && direction === "EXACT_IN")
```

Comment:
`(A && C) || (B && C)` is equivalent to `(A || B) && C`. Therefore, you can simplify this condition like this:

```suggestion
              (isLoading || isRefetching) && direction === "EXACT_IN"
```

Reply:
Thanks, that's a great tip.